ci/pipeline.yml

```diff
@@ -379,10 +336,39 @@ jobs:
                 PR_BASE: master
                 PR_ORG: cloudfoundry
                 PR_LABEL: run-ci
-
                 GCP_SERVICE_KEY: ((gcp.service_key))
                 GITHUB_COM_TOKEN: ((github.access_token))
 
+  - name: build-haproxy-testflight-image
+    public: true
+    serial: true
+    plan:
+      - in_parallel:
+          - get: git-ci
+            trigger: true
+          - get: daily
+            trigger: true
+          - get: docker-cpi-image
+            trigger: true
+      - task: create-daily-tag
+        image: docker-cpi-image
+        config:
+          outputs:
+            - name: tag
+          platform: linux
+          run:
+            path: sh
+            args:
+            - -exc
+            - date +%Y%m%d.%H%M%S | tee tag/today
```

Comment:
This would result in `20241125.081050`, right? Can we make the date a bit easier to read, e.g. `2024.11.25-081050`?